packages/runtime/lib/worker/app.js

```diff
@@ -208,6 +209,19 @@ class PlatformaticApp extends EventEmitter {
     )
     process.exit(1)
   }
+
+  #wrapStackable (stackable) {
+    const newStackable = {}
+    for (const method of Object.keys(defaultStackable)) {
+      newStackable[method] = (...args) => {
```

Comment:
This will make the check at runtime, which I don't think is something we want.
I would go for:

```js
newStackable[method] = stackable[method] ? stackable[method].bind(stackable) : defaultStackable[method]
```

Reply:
done